lightkurve/io/tasoc.py

```diff
@@ -0,0 +1,48 @@
+"""Reader for A PSF-Based Approach to TESS High Quality Data Of Stellar Clusters (PATHOS) light curve files.
+
+Website: https://archive.stsci.edu/hlsp/pathos
+A product description file wasn't obvious on the MAST website
+"""
+from ..lightcurve import TessLightCurve
+from ..utils import TessQualityFlags
+
+from .generic import read_generic_lightcurve
+
+
+def read_tasoc_lightcurve(filename,
+                            flux_column="FLUX_RAW",
+                            quality_bitmask="default"):
```

Comment:
The `quality_bitmask` isn't being used to alter the light curve in any way below, so we ought to remove it from the parameter list for now.

```suggestion
def read_tasoc_lightcurve(filename, flux_column="FLUX_RAW"):
```

Reply:
Done